### PR TITLE
Prettify.

### PR DESCRIPTION
**Pull request by yavorsky**

### CODE REVIEW

**Comment by danez** [comment]
Should we enable prettifier?

**Comment by yavorsky** [comment]
@danez Yeah, I think we could prettify not separate fragments, but all the stuff. The only thing - we must be consistent with babel and babylon prettify branches.

**Comment by xtuc** [comment]
Could we import the rules defined in https://github.com/babel/babel/pull/5413/files#diff-1dc6ee56b778cd91e0327b52aaeaa1b9? We will then include theses rules by default in babel-eslint-config.

**Comment by yavorsky** [comment]
@xtuc Sure 👍 
Also disabled "arrow-parens" and "indent" to prevent errors after prettier.

File: package.json [hunk 1]
```diff
@@ -81,5 +84,19 @@
...
+  "lint-staged": {
+    "src/*.js": [
+      "prettier --trailing-comma all --write \"src/*.js\"",
+      "git add"
+    ],
+    "scripts/*.js": [
+      "prettier --trailing-comma all --write \"scripts/*.js\"",
+      "git add"
+    ],
+    "test/*.js": [
```
Comment (existentialism) [inline package.json]: Probably want to exclude `expected.js`?

File: package.json [hunk 2]
```diff
@@ -81,5 +84,19 @@
     "plugins": [
       "transform-flow-strip-types"
     ]
+  },
+  "lint-staged": {
+    "src/*.js": [
+      "prettier --trailing-comma all --write \"src/*.js\"",
```
Comment (existentialism) [inline package.json]: Can omit the `\"src/*.js\"` as --write will replace

Comment (existentialism) [inline package.json]: Also, can probably combine these to:

```json
{
  "lint-staged": {
    "{src,scripts}/*.js": [
      "prettier --trailing-comma all --write",
      "git add"
    ]
  }
}
```

File: package.json (hunk 1 above)
Comment (yavorsky) [inline package.json]: @existentialism `test/*.js` will cover just files inside script dir, but not below test directory, so `fixtures`/`debug-fixtures` shouldn't be affected.

Comment (existentialism) [inline package.json]: :+1:

File: package.json (hunk 2 above)
Comment (yavorsky) [inline package.json]: @existentialism thanks 👍

**Comment by existentialism** [comment]
@yavorsky mind bumping to latest prettier and rerunning?

**Comment by yavorsky** [comment]
@existentialism 👍

**Pull request merged**